pantheon_content_publisher.install

```diff
@@ -30,3 +30,13 @@ function pantheon_content_publisher_install(): void {
   }
 
 }
+
+/**
+ * Deletes the Pantheon document delete action configuration.
+ */
+function pantheon_content_publisher_update_10001() {
```

Comment:
Shouldn't the number here be:

- 10: Compatible with Drupal 10 or later
- 01: Major version of this module is 1.x
- 1: Sequence number, 1 for the first hook.

So, 10011. Probably works fine as 10001 too, but convention wuld be 10011.